web/cobrands/fixmystreet/fixmystreet.js

```diff
@@ -425,6 +425,8 @@ $.extend(fixmystreet.set_up, {
                 $category_meta.empty();
             }
         });
+
+        fixmystreet.run(fixmystreet.set_up.update_problem_fields($(this).data('role'), $(this).data('body'), args));
```

Comment:
This will not work, because you are passing the results of a function to `fixmystreet.run` – the function by this point has already been called. You need to pass a function reference, be it just a name, or an anonymous function. I'd suggest just checking for `fixmystreet.hooks` as in my last review.